src/components/Dashboard.tsx

```diff
@@ -85,17 +95,22 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
   )
 }
 
-const FilterComponent: React.FC<{ template: TemplateDetails; filter: Filter }> = ({
-  template,
-  filter,
-}) => {
+const FilterComponent: React.FC<{
+  template: TemplateDetails
+  filter: Filter
+  setFiltersCount: (setState: Function) => void
```

Comment:
Sorry, I put you wrong with the type here, this doesn't compile.  For now, just use `setFiltersCount: Function`